lib/types/operation.js

```diff
@@ -242,6 +244,10 @@ Operation.prototype.getType = function (param) {
         return ref;
       }
     } else {
+      // If inline schema, we add it our interal hash -> which gives us it's ID (int)
+      if(schema.type === 'object') {
+        return 'inline#'+ this.addInlineModel(schema);
```

Comment:
Nah, we actually print that when you look at the Model Schema in the response.  Here is a test showing this: https://github.com/swagger-api/swagger-js/blob/master/test/models.js#L313